### PR TITLE
Ft user send mail

### PR DESCRIPTION
**Pull request by neelxie**
[#164605537] Send email to individuals
[#164605796] Save draft or delete draft
#### You can check it out [here](https://neelxie.github.io/epic-mail/UI/compose.html)
@bekeplar @jennizalwango @alexxsanya @Wabuluka @kyakusahmed @emmanuelaaron

### CODE REVIEW

**Comment by Emmanuelaaron** [comment]
So far so good,
I love the responsiveness of your templates

File: UI/assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    margin-bottom: 15px;
+    font-family: Arial, Helvetica, sans-serif;
+    color: rgb(3, 0, 15);
+}
+.input-form input[type="submit"]:hover{
+    background-color: #4772cf;
+    color: white;
+}
+.main{
+    /* display: flex; */
```
Comment (byarustev) [inline UI/assets/css/styles.css:70]: remove this comment

File: UI/assets/css/form.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    text-decoration: none;
+    color: white;
+}
+/* change link text color */ 
+a:hover{
+    color: #ff652f;
+}
+.mylink a:hover{
+    color: rgb(250, 7, 7);
+    /* background-color: white; */
```
Comment (byarustev) [inline UI/assets/css/form.css:135]: remove this comment

**Review by byarustev — CHANGES_REQUESTED** [review]

**Comment by bekeplar** [comment]
Kill the commented code from your code

**Comment by neelxie** [comment]
Thanks, all fixed now.

**Pull request closed without merging**